app/code/Magento/ImportExport/Model/Import/Source/FileParser/UnsupportedPathException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\ImportExport\Model\Import\Source\FileParser;
+
+class UnsupportedPathException extends \RuntimeException
+{
+    private $path;
+
+    public function __construct($path = '', $message = '')
```

Comment:
looks like both of parameters should be mandatory, but not optional 

because creating object with default params , which should be okay, you create a message 
'Path "" is not supported